actions/translation-cursor-deploy/main.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Custom Cursor Deploy GitHub action
```

Comment:
We should have a test here documenting the behaviour and all the corner cases. You should be able to reuse the setup from other actions.